iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/oozie_app/workflow.xml

```diff
@@ -482,6 +482,22 @@
                     <name>patentServiceEndpointOpsUriRoot</name>
                     <value>expectedOpsUriRoot</value>
                 </property>
+				<property>
+					<name>affiliation_matching_number_of_emitted_files</name>
+					<value>1</value>
+				</property>
+				<property>
+					<name>documentsclassification_main_number_of_partitions</name>
+					<value>1</value>
+				</property>
+				<property>
+					<name>referenceextraction_patent_main_number_of_emitted_files</name>
```

Comment:
Could we make it shorter, e.g. like `referenceextraction_patent_number_of_emitted_files`?